dev/breeze/src/airflow_breeze/utils/github.py

```diff
@@ -54,6 +54,13 @@ def download_file_from_github(tag: str, path: str, output_file: Path, timeout: i
     if not get_dry_run():
         try:
             response = requests.get(url, timeout=timeout)
+            if response.status_code == 403:
+                get_console().print(
+                    f"[error]The {url} is not accessible.This may be caused by either of:\n"
+                    f"   1. network issues or VPN settings\n"
+                    f"   2. Github rate limiting limit"
```

Comment:
The double limit looks weird (not a native english speaker here so I might be wrong)

```suggestion
                    f"   2. Github rate limit"
```